src/modelAnalysis/subspaces/checkScaling.m

```diff
@@ -0,0 +1,308 @@
+function [solverRecommendation, scalingProperties] = checkScaling(model, estLevel, printLevel)
+% checks the scaling of the stoichiometric matrix and provides a recommendation on the precision of the solver
+%
+% USAGE:
+%
+%     [solverRecommendation, scalingProperties] = checkScaling(model, estLevel, printLevel)
+%
+% INPUTS:
+%
+%    model:                         COBRA model structure
+%
+% OPTIONAL INPUTS:
+%
+%    estLevel:                      level of estimation: `crude`, `medium`, `fine` (default)
+%    printLevel:                    verbose level (default: 1). Level 0 is quiet.
+%
+% OUTPUTS:
+%
+%    solverRecommendation:          estimation of precision (string, `double` or `quad`, default: `double`)
+%    scalingProperties:             structure with properties of scaling
```

Comment:
Yes, that could be an idea. I will implement that based on the table in `initCobraToolbox`.